zipline/utils/numpy_utils.py

```diff
@@ -142,8 +153,8 @@ def busday_count_mask_NaT(begindates, enddates, out=None):
     if out is None:
         out = empty(broadcast(begindates, enddates).shape, dtype=float)
 
-    beginmask = (begindates == np_NaT)
-    endmask = (enddates == np_NaT)
+    beginmask = (begindates == NaT)
+    endmask = (enddates == NaT)
```

Comment:
`busday_count` only supports `datetime64[D]`, so this should only ever be `NaTD`:

```
In [6]: busday_count(arange(0, 3, dtype='datetime64[ns]'), arange(0, 3, dtype='datetime64[ns]'))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-3e0be598dbd6> in <module>()
----> 1 busday_count(arange(0, 3, dtype='datetime64[ns]'), arange(0, 3, dtype='datetime64[ns]'))

TypeError: Iterator operand 0 dtype could not be cast from dtype('<M8[ns]') to dtype('<M8[D]') according to the rule 'safe'

In [7]: busday_count(arange(0, 3, dtype='datetime64[D]'), arange(0, 3, dtype='datetime64[D]'))
Out[7]: array([0, 0, 0])
```